addons/web_editor/views/snippets.xml

```diff
@@ -395,11 +395,13 @@
          data-selector="img, .media_iframe_video, span.fa, i.fa"
          data-exclude="[data-oe-xpath], a[href^='/website/social/'] > i.fa, a[class*='s_share_'] > i.fa">
         <we-row string="Media">
-            <we-button class="o_we_bg_brand_primary" data-replace-media="true" data-no-preview="true">Replace</we-button>
-            <we-button class="fa fa-link"
+            <we-button class="o_we_bg_brand_primary text-center" data-replace-media="true" data-no-preview="true">Replace</we-button>
```

Comment:
I don't think this change is needed, I think you already forced centering in CSS ?